html/elements/isindex.json

```diff
@@ -0,0 +1,57 @@
+{
+  "html": {
+    "elements": {
+      "isindex": {
```

Comment:
This also has two attributes: prompt and action. Do we want to add them?

Reply:
Yes, fixed.